packages/data-flow/src/orchestrator.ts

```diff
@@ -187,6 +198,31 @@ export class Orchestrator {
         });
     }
 
+    /**
+     * Sometimes the TimestampsUpdated event is part of the _initialize() function of a strategy.
+     * In this case, the event is emitted before the PoolCreated event. We can safely ignore the error
+     * if the PoolCreated event is present in the same block.
+     */
+    private shouldIgnoreTimestampsUpdatedError(
+        error: Error,
+        event: ProcessorEvent<ContractName, AnyEvent>,
+    ): boolean {
+        if (
+            (error instanceof RoundNotFound || error instanceof RoundNotFoundForId) &&
+            (event?.eventName === "TimestampsUpdated" ||
+                event?.eventName === "TimestampsUpdatedWithRegistrationAndAllocation")
+        ) {
```

Comment:
This is just code style, no need to address it if you don't want to but what do you think about extracting those conditions into their own variables?
```suggestion
        const canIgnoreErrorClass = error instanceof RoundNotFound || error instanceof RoundNotFoundForId);
        const canIgnoreEventName = event?.eventName === "TimestampsUpdated" ||
                event?.eventName === "TimestampsUpdatedWithRegistrationAndAllocation";
                
        if (canIgnoreErrorClass && canIgnoreEventName) {
```

Reply:
[c230650](https://github.com/defi-wonderland/grants-stack-indexer-v2/pull/52/commits/c23065046701685ab016abbd3b3fb21f4dd124b0)